packages/react-ui-kit/demo/Demo.jsx

```diff
@@ -153,6 +155,53 @@ class Demo extends React.PureComponent {
0XLolsmoZMmTlo76V6MbYwkpikWxHJj18bej7OPy4/7ZTJxctzk6smP493i58WfS2AkzEzYnfAmMSRxZeLdJIckRVJrsk7ytOS65LcpoSlrUrqmjJsyf8qlVJNUcWpzGiktOW1H2sDUsKnrp/ZM85xWOu3GdPvpRdMvzDCZkTvj6EydmbyZB9IJ6Snpu9M/8mJ5tbyBDG5GTUY/n8PfwH8mCBasE/QKA4RrhE8yAzLXZD7NCsham9UrChJViPrEHHGV+EV2ZPaW7Lc5sTk7c4ZyU3L35pHz0vMOS/QlOZLTs8xnFc3qlDpLS6Vd+X756/P7ZVGyHXJEPl3eXGAAN+xtCgfFt4qHhYGF1YXvZifPPlCkVyQpapvjNGfZnCfF4cU/zsXn8ue2zrOct3jew/ns+VsXIAsyFrQutF5YsrBnUcSiXYupi3MW/7rEdcmaJa+/SfmmpcSsZFFJ97cR39aXapfKSm9+5//dlqX4UvHS9mUeyzYu+1wmKLtY7lpeUf5xOX/5xe/dvq/8fmhF5or2lV4rN68irpKsurE6aPWuNXpritd0r520tnEda13ZutfrZ66/UDG+YssG6gbFhq7K6MrmjTYbV238WCWqul4dUr23xrRmWc3bTYJNVzYHb27YYralfMuHH8Q/3NoasbWx1q62YhtxW+G2x9uTt5/70efHuh0mO8p3fNop2dm1K37X6TrvurrdprtX1qP1ivrePdP2dPwU+lNzg0vD1r3MveX7wD7Fvt9/Tv/5xv6o/a0HfA40HLQ9WHOIcaisEWmc09jfJGrqak5t7jw88XBri3/LoV/G/rLziOWR6qOGR1ceox4rOTZ0vPj4wAnpib6TWSe7W2e23j015dS105NPt5+JOnP+bPjZU+fY546fDzh/5ILfhcMXfS42XfK61Njm2XboV89fD7V7tTde9r7c3OHb0dI5ofPYlaArJ6+GXj17jXvt0vWY6503km7cujntZtctwa2nt3Nvv7hTeGfw7qJ7hHtl93XvVzwwfVD7L8d/7e3y6jr6MPRh26OER3e7+d3PfpP/9rGn5DH9ccUTiyd1T92fHukN7+34fervPc+kzwb7Sv/Q+6PmucPzg38G/9nWP6W/54XsxdDL5a+MX+18Pf5160DcwIM3eW8G35a9M363673P+3MfUj48GZz9kfSx8pPjp5bPUZ/vDeUNDUl5Mp5qK4DBgWZmAvByJwD0VLh36ACAOlV9zlMJoj6bqhD4T1h9FlSJFwA7gwFIWgRANNyjbIbDFmIavCu36onBAPXwGBkakWd6uKu5aPDEQ3g3NPTKDABSCwCfZENDg5uGhj5th8neBuBEvvp8qRQiPBv8YKxEbTd1wdfyb6InfkrYXYKdAAAACXBIWXMAABYlAAAWJQFJUiTwAAABm2lUWHRYTUw6Y29tLmFkb2JlLnhtcAAAAAAAPHg6eG1wbWV0YSB4bWxuczp4PSJhZG9iZTpuczptZXRhLyIgeDp4bXB0az0iWE1QIENvcmUgNS40LjAiPgogICA8cmRmOlJERiB4bWxuczpyZGY9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkvMDIvMjItcmRmLXN5bnRheC1ucyMiPgogICAgICA8cmRmOkRlc2NyaXB0aW9uIHJkZjphYm91dD0iIgogICAgICAgICAgICB4bWxuczpleGlmPSJodHRwOi8vbnMuYWRvYmUuY29tL2V4aWYvMS4wLyI+CiAgICAgICAgIDxleGlmOlBpeGVsWERpbWVuc2lvbj4xNjwvZXhpZjpQaXhlbFhEaW1lbnNpb24+CiAgICAgICAgIDxleGlmOlBpeGVsWURpbWVuc2lvbj4xNjwvZXhpZjpQaXhlbFlEaW1lbnNpb24+CiAgICAgIDwvcmRmOkRlc2NyaXB0aW9uPgogICA8L3JkZjpSREY+CjwveDp4bXBtZXRhPgr0f6jpAAAAHGlET1QAAAACAAAAAAAAAAgAAAAoAAAACAAAAAgAAAFq4smoVQAAATZJREFUOBF0kEtLAlEYhp9vxtEudlFp0aZN0CoobFQo6k+0adeynxBtWgr9wJJoEWVBaU6o4DTTXPrOpIiRB85ZnPM97+VIo95I+W8JmAdrlFBunmBXNxAvINUdPw4I73t8P3hI3Z0j4Fgk3RGliyr5020ibwR6JzkbydtYIogfzRFwhNjzWTncYvnSJfJDRCzNo5kSPdM026JzUnNrMxXEtoiDiMLqEus3x8RFG8KECY+plhhEsGwjcPBXQCeGEeXrI9ivkAwDjT3+EJNBWU2vsMX73fOsgJjeHZ+1813yZzvE/S+dHEfPrDW6itiOTef2icHb51TAwAZY3Ntk9apGqDXUSO1+42bWE7jVZvDaI1csIK6poHGSMMYpOJSa2rviQBBPYQUzZ/39j9YL/XZX4QVSFf8BAAD//1tgTEwAAAGDSURBVE2QP0scYRCHn9lb1zv1/siB4pFWNF0ULxdsPMFGmzRBJEXIV9BGCHb5AiEQJIWtfaoUqYTrUoQkKgYCCShYKJyR6O6e++44L2ckU77v/J6ZZ2RutqmEAXR71DYfI0/G0MsUQgHlrpRgIKT79Zju2TnhcAnEPkWQuVZL9Sym/GyK6MU07jKBgoV9eYBTpDpIvHvExY9TsuUJJHUQWNjaZObhjJYmRylvtbhRh+QW8nQEdTlSsfCHn/S2D8laNeKVcaRn77a0L2k+amrt9TyuYWslmZHtVQ2dWZOf/PEX6c4BMhTh6iHJasOz7/Vk8c1LlXYD/ZMghaCv7SdXiySffpO8/47UBg2oaCTEzx+gpQLkpuYVlr680sy85Z+3OVOJSPdOSN59g3LkN7WJFjBIvNYgHy/CjQ3xgIXOhva9rclTzTntWPithUcG+uv6u3i1q5z06RhuqgJ2SH8Hae+t+4vZAMMVA3qfT0m39yGyNf9z9QD560jbddx8Hb3uA24BdMgUmcDgSrIAAAAASUVORK5CYII='
```

Comment:
What do you think about moving these to files?

Reply:
Just for the demo..